pkg/operator/sync.go

```diff
@@ -215,31 +216,37 @@ func (c *csiSnapshotOperator) syncProgressingCondition(instance *operatorv1.CSIS
 		// Not reachable in theory, but better to be on the safe side...
 		progressing = operatorv1.ConditionTrue
 		progressingMessage = "Waiting for Deployment to be created"
+		reason = "Deploying"
 
 	case deployment.Generation != deployment.Status.ObservedGeneration:
 		progressing = operatorv1.ConditionTrue
 		progressingMessage = "Waiting for Deployment to act on changes"
+		reason = "Deploying"
```

Comment:
How is `Deploying` different than `DeployingController`?

Reply:
My thoughts were that `DeployingController` indicates that the deployment has already been updated, and we're simply waiting on the pods to roll out (in whatever form). `Deploying` indicates that we're still waiting on the deployment to be created/updated.

If you think there's no need to differentiate these two then I can update it to be the same reason.

Reply:
In this particular case, the Deployment API object has been already created / updated, but we're waiting for Kubernetes controller-manager to act on it. IMO it's better if OCP uses `Deploying` for everything.

Reply:
I've updated this PR and the library-go one to use `Deploying` for all of these states.